docs/_sources/auto_examples/plot_2D_simulation_example.rst.txt

```diff
@@ -554,9 +554,9 @@ randomization.
 
 .. rst-class:: sphx-glr-timing
 
-   **Total running time of the script:** (1 minutes 4.127 seconds)
+   **Total running time of the script:** (1 minutes 9.317 seconds)
```

Comment:
Why did the computation time increase so much ?

Reply:
I think it's part of the variability of the test. It's a difference of 5 seconds over 1 min; for me it's negligible.
From my mistake, it's just a rerun of the documentation; thus, there is not really an indication of the difference in time.

Reply:
@bthirion